unit/testing-utils/use_catch.h

```diff
@@ -36,4 +36,7 @@ Author: Michael Tautschnig
 #include <util/pragma_pop.def>
 #endif
 
+/// Add to the end of test tags to mark a test that is expected to fail
+#define XFAIL "[.][shouldfail]"
```

Comment:
Why?

Reply:
From the commit message:
> These tests won't be run by default (de-cluttering the output with expected failures). When run, they will fail if they pass

In previous PR Owen / Micahel complained that shouldfail was misleading. They _shouldn't_ fail, in the perfect world, but they do due to known bug. We felt expected fail (or XFAIL as python calls them) was clearer. Can change to `EXPECTED_FAIL` if you'd prefer?